src/org/team1540/robot2017/RobotUtil.java

```diff
@@ -0,0 +1,18 @@
+package org.team1540.robot2017;
+
+public class RobotUtil {
+	public static double deadzone(double input, double range) {
+		if (Math.abs(input)<range) {
+			return 0.0;
+		}
+		return input;
+	}
+	public static double limit(double input, double max, double min) {
```

Comment:
No change needed unless you see this and have a chance--format this code correctly so that it adheres to the Eclipse style guide. You can Google how to automatically format code in Eclipse.